app/build.gradle

```diff
@@ -1,15 +1,16 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 21
+    compileSdkVersion 22
```

Comment:
It's better to update build tools, library versios, compile SDK and so on before presenting the sample to clients.